config/all-in-one/templates/_helpers.tpl

```diff
@@ -2,11 +2,21 @@
 Specify the config path
 */}}
 {{- define "cp4waiops.configPath" -}}
-  {{- if eq .Values.cp4waiops.version "3.2" }}
+  {{- if eq .Values.cp4waiops.version "v3.2" }}
   {{- printf "config/3.2/cp4waiops" -}}
-  {{- else if eq .Values.cp4waiops.version "3.3" }}
+  {{- else if eq .Values.cp4waiops.version "v3.3" }}
   {{- printf "config/3.3/ai-manager" -}}
   {{- else }}
-  {{- fail "The CP4WAIOps all in one chart only supports release 3.2, 3.3." }}
+  {{- fail "The CP4WAIOps all in one chart only supports release v3.2, v3.3." }}
   {{- end }}
 {{- end -}}
+
+{{- define "eventmanager.configPath" -}}
+  {{- if eq .Values.cp4waiops.version "v3.2" }}
+  {{- printf "config/3.2/cp4waiops" -}}
```

Comment:
Once we get this pr merged, we can address the cross release consistency by changing folder structure for 3.2 release that aligns w/ 3.3.

Reply:
Understand, will be revised when 3.2 is completed